assets/trino/functions.sdf.yml

```diff
@@ -6377,25 +7735,31 @@ function:
   optional-parameters: []
   returns:
     datatype: array<$9>
+  cross-link: https://prestodb.io/docs/current/functions/array.html#zip_with
+  section: array
 ---
```

Comment:
Actually, based on the alphabetic ordering (`zip_with` above, `if` below) I suspect the whole rest of the file is a victim of an unfortunate merge.